.lando.yml

```diff
@@ -15,9 +15,17 @@ tooling:
     cmd:
       - ./vendor/bin/codecept
       - "--env=lando"
+  npm:
```

Comment:
We should only use one tool by default, we have too many projects with a mixture and that's causing inconsistencies. CircleCI is configured to use yarn, so it would make sense to use that.